README.md

```diff
@@ -47,29 +47,22 @@ terraform init --upgrade=true
 terraform apply
 ```
 
-Once the user is created you will need to update the
-[repository's secrets](https://github.com/cisagov/freeipa-server-packer/settings/secrets)
-with the new encrypted environment variables.
-
-```console
-terraform state show module.iam_user.aws_iam_access_key.key
-```
-
-Take the `id` and `secret` fields from the above command's output and create the
-`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables in the
-[repository's secrets](https://github.com/cisagov/freeipa-server-packer/settings/secrets).
-
-You will also need to add one additional repository secret called
-`BUILD_ROLE_TO_ASSUME`.  Here is how to see the ARN that you need to set
-as the value for that secret:
-
-```console
-terraform state show module.iam_user.aws_iam_role.ec2amicreate_role[0] | grep ":role/"
-```
-
-IMPORTANT: The account where your images will be built must have a VPC and
-a public subnet both tagged with the name "AMI Build", otherwise `packer`
-will not be able to build images.
```

Comment:
Is this no longer needed?

Reply:
Sharp eyes, @mcdonnnj!  I accidentally merged in `skeleton-packer/develop` instead of `skeleton-packer-cool/develop`.  That's why the text got clobbered.

Can you look at what is thereafter commit 332604d and let me know if it looks correct now?

Reply:
Good save @mcdonnnj!  🥇

Reply:
Ah, that makes sense. Looks good now 👍